action/Pickup.action

```diff
@@ -65,6 +65,8 @@ string[] trajectory_descriptions
 # The performed grasp, if attempt was successful
 Grasp grasp
 
+# The amount of time in seconds it took to complete the plan
```

Comment:
This seems ambiguous. Is it the time to plan or the time to execute the plan? Maybe reword to "it took to generate a plan"